src/util/logger/Logger.cpp

```diff
@@ -39,7 +41,16 @@ static string get_worker_name() {
 
 void Logger::log(std::string message, const std::string log_type) {
     pthread_t tid = pthread_self();
-    message = "[" + worker_name + " : " + to_string(tid) + "] " + message;
+
+    struct timeval tv;
+    gettimeofday(&tv, NULL);
+    long millis = tv.tv_sec * 1000 + tv.tv_usec / 1000;
+    // TODO: temporary fix only.
```

Comment:
Can you please provide more details why this is temporary fix?

Reply:
The spdlog hangs when forked. This prevents that.

Reply:
Can you please add those details and update the comment?

Reply:
Updated

Reply:
Ok